what_is.md

```diff
@@ -249,19 +256,26 @@ plt.show()
 
 ## A record of genetic ancestry
 
+::::{margin}
```

Comment:
This one collides with the "todo" but I think that is ok as it is a temporary problem.
![Screenshot from 2021-06-22 10-26-11](https://user-images.githubusercontent.com/8552/122900073-47286a80-d344-11eb-95a2-96b9bc7a3690.png)

Reply:
Yep - I'm just sorting that now. Also they don't actually overlay each other, unlike the code boxes.